packages/fast-tooling-react/src/css-editor/editor.props.ts

```diff
@@ -1,25 +1,25 @@
 import { ManagedClasses } from "@microsoft/fast-jss-manager";
 import { CSSEditorClassNameContract } from "./editor.style";
-import { Subtract } from "utility-types";
-import { CSSPositionHandledProps, CSSSpacingHandledProps } from "./";
-import { CSSSpacingClassNameContract } from "./spacing/spacing.style";
-import { CSSPositionClassNameContract } from "./position/position.style";
+import { CSSSpacingValues } from "./spacing";
+import { CSSPositionValues } from "./position";
 
-export type CSSOnChange = (CSS: any) => void;
+export interface CSSEditorValues extends CSSSpacingValues, CSSPositionValues {}
+
+export type CSSOnChange = (CSS: CSSEditorValues) => void;
 
 export interface CSSEditorUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
 
 export interface CSSEditorHandledProps
-    extends Subtract<
-            CSSPositionHandledProps,
-            ManagedClasses<CSSPositionClassNameContract>
-        >,
-        Subtract<CSSSpacingHandledProps, ManagedClasses<CSSSpacingClassNameContract>>,
-        ManagedClasses<CSSEditorClassNameContract> {
+    extends ManagedClasses<CSSEditorClassNameContract> {
+    /**
```

Comment:
minor nit, but I think `onChange` may be more accurate here. Correct me if I'm wrong, but my understanding is that the callback gets called when the user has changed the inputs right? The author still needs to update the input data with the reflected changes for the UI to to reflect the changes, right?

Reply:
That's true, I'll change it back